api/src/app/modules/attachment/attachment.controller.ts

```diff
@@ -76,6 +77,7 @@ export class AttachmentController {
 
   @Get('/file/:id')
   @ApiBearerAuth()
+  @AuthGuardMeta(GUARD_OPTIONS.PUBLIC)
   @ApiOkResponse() 
   async getFileContents(
```

Comment:
This was fixed few days ago as even current production is broken.
Currently, to make it works, it can not be secured for "user" object so I open it to non-secured to make it work (did mention that in previous pr description about this change).
The problem is from frontend (and the code wasn't changed either from old time).
Frontend only triggers a "link" to open and download file (without going throw api request, so does not even attach auth token).
We can have separate ticket to deal with this. Both frontend and backend need to be changed to restrict the file download for only permitted "user". Currently, even the backend does not have logic for using "user" object even if there is a "user".

Reply:
Discussed, will have separate ticket to deal with it both frontend and backend.
TODO comment added.